h2o-cve-2018-0608/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu AS base
```

Comment:
Please specify an ubuntu version (in two years, the ubuntu:latest may not work).